finetuner/tuner/state.py

```diff
@@ -1,10 +1,14 @@
 from dataclasses import dataclass, field
 from typing import Dict
 
+from rich.progress import Progress
+
 
 @dataclass
 class TunerState:
-    """A data container representing containing information on the current run."""
+    """A data container representing information on the current run."""
+
+    progress_bar: Progress = Progress()
```

Comment:
Hm, why do we need this? Does creating a second (independent) progress bar cause any issues?

Reply:
Okay, then let's make the progress bar an attribute of the `Tuner` directly